mobile/src/board/components/BoardView/index.css

```diff
@@ -0,0 +1,16 @@
+:root {
```

Comment:
In general, we avoid arranging files by type (component, hook) and using that type to name folders. Read https://marmelab.com/blog/2015/12/17/react-directory-structure.html for the rationale.